### DISCUSSION TITLE
Kaboom is great

### DISCUSSION THREAD

**Discussion by joshuacurtiss** [replit/kaboom#193] [General]
I've been asking about extra features a lot lately and wanted to just take a moment to say that I love the Kaboom library. The foundation of how it works is so simple and powerful. Creating game objects and adding functionality like a plugin architecture works so well. Enjoying it very much, thank you! 🚀

**Comment by slmjkdbtl** [comment]
Thank you! Feature requests is the most welcome thing here as it'll help kaboom grow, don't let anything stop you from doing that 🙏

**Comment by lajbel** [comment]
kaboom is GOD
